cmd/zetacored/root.go

```diff
@@ -111,7 +130,19 @@ func NewRootCmd() (*cobra.Command, types.EncodingConfig) {
 				return err
 			}
 
-			customAppTemplate, customAppConfig := initAppConfig()
+			// TODO evm: need to check about evm chain id, getting it like this is generally fine, but some commands
```

Comment:
Let's open issues for the TODO that we don't need to address immediately.
We can group them in a EPIC "Cosmos EVM follow-up"

Reply:
this one i will try to address before merging, it was communicated, waiting to see if they can simplify it somehow